templates/mobile/src/amplitude.js

```
@@ -0,0 +1,44 @@
+import { Amplitude } from '@amplitude/react-native';
+import { getVersion, getBundleId, getApplicationName } from 'react-native-device-info';
+import ApollosConfig from '@apollosproject/config';
+
+const ampInstance = Amplitude.getInstance();
+ampInstance.init(ApollosConfig.AMPLITUDE_KEY);
+
+export const track = ({ eventName, properties = null }) => ampInstance.logEvent(eventName, properties);
+
+export const identify = () => {
+    const data = client.readQuery({
```

this was the linter giveaway. `client` and `gql` here are undefined

---

ah thanks! we swallow the error so I never saw it 😬